WordPress/Classes/ViewRelated/System/FancyAlertViewController+QuickStart.swift

```diff
@@ -25,6 +26,11 @@ extension FancyAlertViewController {
             controller.dismiss(animated: true, completion: nil)
         }
 
+        let neverButton = ButtonConfig(Strings.neverText) { controller, _ in
+            UserDefaults.standard.quickStartWasDismissedPermanently = true
+            controller.dismiss(animated: true, completion: nil)
```

Comment:
The `completion` parameter defaults to `nil`; can we replace
```
controller.dismiss(animated: true, completion: nil)
```
with:
```
controller.dismiss(animated: true)
```
?

I recognize this following the convention on lines 22 & 26, so I'm receptive to your thoughts on this idea.